react-components/hello-world.md

```diff
@@ -0,0 +1,386 @@
+
+In this curriculum, we will rarely touch `src/index.js`. Additionally, there are other lines of code in this file that is worth exploring with bigger projects.
+
+Instead, we'll focus on leveraging the `App` component.
+
+### Exploring `App.js`
+
+Consider the contents of `src/App.js`.
+
+```js
+import logo from "./logo.svg";
+import "./App.css";
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
+}
+
+export default App;
+```
+
+From reading `src/App.js`, answer the questions below.
+
+<!-- Question 5 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: checkbox
+* id: 429bf219
+* title: Hello, World!
+##### !question
+
+`src/App.js` imports several files. Select the files that are imported.
+
+##### !end-question
+##### !options
+
+* `React`
+* `'./index.js'`
+* `'./App.js'`
+* `App` from `'./App'`
+* `logo` from `'./logo.svg'`
+* `'./App.css'`
+
+##### !end-options
+##### !answer
+
+* `logo` from `'./logo.svg'`
+* `'./App.css'`
+
+##### !end-answer
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 6 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: multiple-choice
+* id: cde06659
+* title: Hello, World!
+##### !question
+
+There is an object named `App`. `App` is a...
+
+##### !end-question
+##### !options
+
+* variable
+* function call
+* class
+* function
+
+##### !end-options
+##### !answer
+
+* function
+
+##### !end-answer
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 7 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: checkbox
+* id: 6d8a4864
+* title: Hello, World!
+##### !question
+
+Check all options that are true about `src/App.js`.
+
+##### !end-question
+##### !options
+
+* `App` is a function, and it returns something
+* `App` is an arrow function
+* `logo` is not actually used in this file
+* The `export default App;` statement is used to make `App` available to other files
+
+##### !end-options
+##### !answer
+
+* `App` is a function, and it returns something
+* The `export default App;` statement is used to make `App` available to other files
+
+##### !end-answer
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+The `App` function defines a component named `App`.
+
+The responsibility of `App` function is to return _how_ to render the `App` component whenever the `App` component is declared and used.
+
+### !callout-info
+
+## Explore!
+
+Now is a great time to explore our webapp. We can change the headers and text, and observe how they get reflected in our app.
```

Comment:
Git's still a thing that we can encourage students to practice. Since it gets set up by default, they can start committing/reverting right from the get go!

```suggestion
Now is a great time to explore our webapp. We can change the headers and text, and observe how any changes get reflected in our browser.

<br />

Notice that `create-react-app` automatically initialized a Git repository in our directory. So if something breaks during our exploration we can always get back to the initial version of the files using our Git skills!
```